privacyidea/lib/caconnectors/localca.py

```diff
@@ -468,9 +468,13 @@ def get_templates(self):
         """
         content = {}
         if self.template_file:
-            with open(self.template_file, 'r') as content_file:
-                file_content = content_file.read()
-                content = yaml.safe_load(file_content)
+            try:
+                with open(self.template_file, 'r') as content_file:
+                    file_content = content_file.read()
+                    content = yaml.safe_load(file_content)
+            except (FileNotFoundError, PermissionError):
+                log.warning("Template file {0!s} for {1!s} not found or "
+                            "not permitted.".format(self.template_file, self.name))
```

Comment:
If the admin is really confused about this, "oh, but this file exists!", then please do as we do in a lot of other places and also add a 

    log.debug('{0!s}'.format(traceback.format_exc()))

Need to import ``traceback`` before.

This way the admin can turn on debug level and knows, why the file can not be read.

Reply:
done

Reply:
Oups, we started to use 

   log.debug(u'{0!s}'...

if the traceback contains a hint to the problem, like the filename, we could have a filename with a non-ascii character and then the log.debug would cause another exception.
I will add the u"".